jujugui/static/gui/src/app/jujulib/test-baker-factory.js

```diff
@@ -0,0 +1,46 @@
+/* Copyright (C) 2016 Canonical Ltd. */
+
+'use strict';
+
+chai.config.includeStack = true;
+chai.config.truncateThreshold = 0;
+
+describe('jujulib bakery factory', function() {
+
+  var fakeBakery = function(cfg) {
+    this.serviceName = cfg.serviceName;
+  };
+
+  fakeBakery.prototype.clearCookie = sinon.stub();
+
+  var makeFactory = function() {
```

Comment:
const, but really this helper would not be required if we don't require cfg at all, as mentioned above.